config/settings.yml

```diff
@@ -74,7 +74,6 @@ DOI:
 HARVESTER:
   LOG: log/all_sources_harvester.log
   USE_MIDDLE_NAME: true
-  USE_FIRST_INITIAL: false
```

Comment:
this configuration parameter is no longer needed since we will now selectively decide if we want to use the first initial or not